[EN] Creates install lesson for phoenix (#2723)
elixirschool/elixirschool: lessons/en/phoenix/installing.md

%{
  version: "1.0.0",
  title: "Installing Phoenix",
  excerpt: """
  With some simple steps you can install and create your first phoenix project
  """
}
---

## Getting Started

### Some considerations

> Before aaaaaaaaaaaaaaa creating a Phoenix project, you will need to install elixir in your machine. The latest version of phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
> 

Reviewer: ```suggestion
Before creating a new Phoenix app we need to ensure we have compatible versions of Elixir installed. The latest version of Phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
```

In general we prefer to use "we" in places of "you" or "I" so it the learning process feels collaborative and less directed at someone or about someone. Could you work through the lesson updating those references?

Author: Yeah, thanks for reviewing the PR @doomspork . I will update the PR next weekend!

> You will also need some database to run your application,  if you are just getting started with Phoenix, we recommend you to install PostgreSQL. You can check the [installation guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides) to install it to your OS.
> 

Reviewer: ```suggestion
In addition we'll need a database to run our application, we recommend using PostgreSQL. You can check the [installation guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides) to install it to your OS.

```
